### DISCUSSION TITLE
how can i redirect some path to other

### DISCUSSION THREAD

**Discussion by zhoutengshen** [nuxt/nuxt#17673] [Questions]
how can i redirect some path to other like VueRouter
eg:
![image](https://user-images.githubusercontent.com/26021548/165797306-4465971f-41aa-4bf6-ae5f-366672fddcbd.png)

Because I promoted the home page to my friends, but I modified the page structure of nuxt page, which is now [page/ newhome]. I hope my friend's visit to the [/home] page is equivalent to visiting /newhome[]

**Comment by necatikcl** [comment]
You can do it with middlewares:

```js
definePageMeta({
	middleware: () => '/home'
})
```

https://v3.nuxtjs.org/guide/directory-structure/middleware/
